frontend/src/views/eligibility/CheckEligibility.vue

```diff
@@ -134,7 +135,7 @@ export default {
         this.searchOk = true
         this.alertStore.setAlert({ message: this.result.message, type: this.result.status })
       } catch (err) {
-        this.alertStore.setErrorAlert(err)
+        handleServiceError(err, this.alertStore, this.$router)
```

Comment:
Should we just add these to all the pages now? Rather than refactor others later